pipenv/environments.py

```diff
@@ -88,3 +88,5 @@
 SESSION_IS_INTERACTIVE = bool(os.isatty(sys.stdout.fileno()))
 
 PIPENV_SHELL = os.environ.get('SHELL') or os.environ.get('PYENV_SHELL')
+
+PIPENV_CACHE_DIR = os.environ.get('PIPENV_CACHE_DIR', user_cache_dir('pipenv'))
```

Comment:
Does the cache dir get cleared on reboot?  Or is it persistent?

Reply:
It persists across reboots